ceno_zkvm/src/bin/e2e.rs

```diff
@@ -282,9 +288,11 @@ fn run_inner<
     let vk_bytes = bincode::serialize(&vk).unwrap();
     fs::write(&vk_file, vk_bytes).unwrap();
 
-    let verifier = ZKVMVerifier::new(vk);
-    verify(&zkvm_proof, &verifier).expect("Verification failed");
-    soundness_test(zkvm_proof, &verifier);
+    if !skip_verify {
```

Comment:
How about we define a new e2e stage "PrepProof"? I think this design fit more natural in original e2e flow.

Take `run_elf_inner` as example
```rust
run_elf_inner::<E, PCS, P>(args, elf_path, Checkpoint::PrepSanityCheck, skip_verify)?;
```  

here 3rd and 4th got some subset relation, somehow indicated one of them is redundant

Reply:
oh! good point!